gcp-repo/metadata.product

```diff
@@ -9,7 +9,7 @@
   .p2.inf).  p2 generates unique qualifiers for categories, so it would be
   best for our non-release builds to have unique versions too.
 -->
-<product uid="com.google.cloud.tools.eclipse.dist" version="1.3.0.qualifier" useFeatures="true" includeLaunchers="false">
+<product uid="com.google.cloud.tools.eclipse.dist" version="1.3.1" useFeatures="true" includeLaunchers="false">
```

Comment:
I think the release instructions are not up to date. I remember @briandealwis and I decided that we retain `.qualifier` here and drop it temporarily when actually building binaries in a local tag branch (which is indeed not ideal). I'll update the release instructions.

Reply:
We should retain `.qualifier` in this PR. I have updated the release instructions accordingly:

> 5. Checkout the tag: `git checkout <tag>`.
> 6. Use `git status` to verify there are no local changes. Make sure you see "nothing to commit, working directory clean".
> 7. Drop the `.qualifier` in `gcp-repo/metadata.product` (e.g., `version="1.3.0.qualifier"` to `version="1.3.0"`) and use the exact release version ([#2211](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/2211)). Do not commit. `git status` should show that `gcp-repo/metadata.product` is the only file that changed.

@briandealwis we are good with this, aren't we?

Reply:
That's what we did before.  Perhaps we should change it slightly and:
  - increment `1.3.1.qualifier`
  - commit & push to _master_
  -  tag 1.3.1
  - drop the `.qualifier` and do the actual build

Otherwise `master` stays at `1.3.0.qualifier` and we'll need to fix that up separately.

Reply:
That's exactly what I am saying. This PR will change the version to `1.3.1.qualifier`.